### DISCUSSION TITLE
.vimrc in nvchad

### DISCUSSION THREAD

**Discussion by khodekhadem** [NvChad/NvChad#1884] [Q&A] [answered]
hi, i just want my old .vimrc to work in nvcahd but i don't know how 

---->.vimrc
set termbidi
set tabstop=4
set shiftwidth=4
set softtabstop=4
set expandtab
set wildmode=longest,list,full
set wildmenu
set autoindent
set rnu
set ic

**Comment by siduck** [comment, marked as answer]
hi there @khodekhadem , use vim.opt . 

![image](https://user-images.githubusercontent.com/59060246/229447172-6bc707c5-04b8-4cf8-ada2-7006de506eb3.png)


BTW some of those options are already configure in our config. check core/init.lua
